src/current/_includes/releases/v23.1/v23.1.9.md

```diff
@@ -0,0 +1,244 @@
+- This patch fixes an [optimizer](../v23.1/cost-based-optimizer.html) costing bug introduced in [v23.1](../releases/v23.1.html) which may cause a query whose best-cost query plan is a string of [lookup joins](../v23.1/joins.html#lookup-joins) with [`REGIONAL BY ROW`](../v23.1/table-localities.html#regional-by-row-tables) tables, one after the other in sequence, to not pick the most optimal join plan. [#108308][#108308]
+- Previously, using [`IMPORT INTO`](../v23.1/import-into.html) for `DELIMITED DATA` or MySQL imports would error with `column ... does not exist` if it was importing into a [collated string](../v23.1/collate.html) column. [#108286][#108286]
+- Fixed a bug introduced in [v23.1](../releases/v23.1.html) that could cause the precision of some values to be incorrectly truncated for a query with a correlated [subquery](../v23.1/subqueries.html) and an equality between a column from the subquery and the outer query. This applies to types that are "equivalent" but have different precision levels, e.g. [`DECIMAL(10, 0)`](../v23.1/decimal.html) vs `DECIMAL(10, 2)` or `NAME` vs [`CHAR`](../v23.1/string.html). [#108198][#108198]
+- Fixed a bug where [`BEGIN`](../v23.1/begin-transaction.html), [`COMMIT`](../v23.1/commit-transaction.html), [`SET`](../v23.1/set-vars.html), [`ROLLBACK`](../v23.1/rollback-transaction.html), and [`SAVEPOINT`](../v23.1/savepoint.html) statements would not be written to the [execution](../v23.1/logging-overview.html#logging-destinations) or [audit logs](../v23.1/sql-audit-logging.html). [#108411][#108411]
+- Fixed a bug where a session migration performed by `SHOW TRANSFER STATE` would not handle prepared statements that used the [`AS OF SYSTEM TIME` clause](../v23.1/as-of-system-time.html). Users who encountered this bug would see errors such as `expected 1 or 0 for number of format codes, got N`. This bug was present since [v22.2.0](../releases/v22.2.html). [#108548][#108548]
+- Fixed errors on the [Sessions page](../v23.1/ui-sessions-page.html) in the [DB Console](../v23.1/ui-overview.html) when a session's memory usage is zero bytes. [cockroachdb/cockroach#108733][#108733]
+- Fixed a bug introduced in [v22.1](../releases/v22.1.html) that could cause a [join](../v23.1/joins.html) to infinite-loop in rare cases when (1) the join filter is not an equality and (2) no columns from the left input are returned. [#108821][#108821]
+- Fixed an issue with the full scan filter on the [Statements page](../v23.1/ui-statements-page.html) where the filter was always evaluating to false, even if a full scan had occurred. [cockroachdb/cockroach#109254][#109254]
+- Fixed a bug that could cause CPU usage to increase over time. [#109298][#109298]
```

Comment:
Looking at the code, it appears to be somewhere in the metrics jobs code, but it could have been anywhere.  It's mostly an obscure internal Golang thing about "contexts" so I think if someone has expertise/curiosity and wants to read the PR they can (it's a one-line change)